osltweaks.php

```diff
@@ -16,6 +16,35 @@ function osltweaks_civicrm_buildForm($formName, &$form) {
       // include some javascript to finagle the wording in that section.
       CRM_Core_Resources::singleton()->addScriptFile('org.osltoday.osltweaks', 'js/CRM_Contribute_Form_Contribution_Main-isShowCMS.js');
     }
+
+    // Array of Contribution page id for the Country and State rewrite
+    $settings = CRM_Core_BAO_Setting::getItem(NULL, 'com.joineryhq.osltweaks');
+
+    foreach ($settings['page_ids'] as $key => $id) {
+      // Check if contribution page id is match on the $settings and make changes
+      if ($id == $_GET['id']) {
+        if ( $form->elementExists( 'country-1' ) ) {
+          // Remove other Country and show only US
+          $elCountry = & $form->getElement('country-1');
```

Comment:
The `&` isn't needed here, because `getElement` returns a reference. Same for other uses of  `getElement` in this PR.

Reply:
The code will not work if `&` is removed in `$elCountryOptions = & $elCountry->_options;`. So I leave the `&` sign in that line.